A workaround for this error:

Traceback (most recent call last):
  File "/usr/lib/cinnamon-settings/cinnamon-settings.py", line 1838, in side_view_nav
    sidePage.build()
  File "/usr/lib/cinnamon-settings/cinnamon-settings.py", line 744, in build
    scrolledWindow.set_shadow_type(Gtk.ShadowType.IN)
AttributeError: type object 'GtkShadowType' has no attribute 'IN'

Disable the shadow type for now.

--- files/usr/lib/cinnamon-settings/cinnamon-settings.py.orig	2012-09-22 16:47:36.560928739 +0200
+++ files/usr/lib/cinnamon-settings/cinnamon-settings.py	2012-09-22 16:47:47.218674599 +0200
@@ -741,7 +741,7 @@ class ThemeViewSidePage (SidePage):
             iconView.select_path(self.model.get_path(self.active_theme_iter))
         iconView.connect("selection_changed", self.apply_theme )
         scrolledWindow.add(iconView)
-        scrolledWindow.set_shadow_type(Gtk.ShadowType.IN)
+        scrolledWindow.set_shadow_type(Gtk.ShadowType.NONE)
         link = Gtk.LinkButton("http://cinnamon-spices.linuxmint.com/themes")
         link.set_label(_("Get new themes"))    
         cinnamon_theme_vbox.pack_start(link, False, False, 2)
